gradle.properties

```diff
@@ -1,17 +1 @@
 version=4.14.0
-#
-# main dependencies
-#
-hivemq-amazon-kinesis-sdk.version=4.14.0
```

Comment:
We have to update this part also: https://github.com/hivemq/hello-world-extensions-version-updater/blob/39e8320b19c5a004cf3e9411a44b9b7ae7564a7e/update-hello-world-kinesis-extension.sh#L9

Reply:
Created a separate card: https://hivemq.kanbanize.com/ctrl_board/41/cards/17046/details/